style.css

```diff
@@ -306,3 +326,175 @@ input {
   padding: 10px 13px;
   align-self: baseline;
 }
+
+/* Desktop-version */
+@media (min-width: 768px) {
+  /* nav-menu */
+  nav {
+    padding: 12px 150px;
+  }
```

Comment:
- [ ] [Optional] You can add box-shadow to your design to make your navbar look more beautiful and match the Figma design and you can make it fixed too.

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/171422180-4ac102a1-3b34-4bbc-ad89-cd915447576f.png)  | ![image](https://user-images.githubusercontent.com/68796232/171422142-cda3876e-e697-4773-9517-f60e44b2954d.png) |